dbt_subprojects/hourly_spellbook/models/_metrics/fees/metrics_fees_index_daily.sql

```diff
@@ -0,0 +1,57 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'fees_index'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+{% set baseline_date = '2018-01-01' %}
+
+with baseline as (
+    select
+        sum(gas_fees_usd) as baseline_gas_fees_usd -- sum is required due to blockchain being second unique key in source
+    from
+        {{ ref('metrics_gas_fees_daily') }}
+    where
+        block_date = date '{{ baseline_date }}'
```

Comment:
this will filter us down to ethereum only i believe, keeping values lower than anticipated, since other chains don't have data this far back

Reply:
(note: applies to all three fees/txs/transfers spells)